README.md

```diff
@@ -1,37 +1,55 @@
-# haskell-library-template
+# AWS X-Ray Support for hs-opentelemetry
 
-Haskell library template used at Freckle.
+<!-- TODO
+[![Hackage](https://img.shields.io/hackage/v/hs-opentelemetry-awsxray.svg?style=flat)](https://hackage.haskell.org/package/hs-opentelemetry-awsxray)
+[![Stackage Nightly](http://stackage.org/package/hs-opentelemetry-awsxray/badge/nightly)](http://stackage.org/nightly/package/hs-opentelemetry-awsxray)
+[![Stackage LTS](http://stackage.org/package/hs-opentelemetry-awsxray/badge/lts)](http://stackage.org/lts/package/hs-opentelemetry-awsxray)
+-->
 
-## Create your repo
+[![CI](https://github.com/freckle/hs-opentelemetry-awsxray/actions/workflows/ci.yml/badge.svg)](https://github.com/freckle/hs-opentelemetry-awsxray/actions/workflows/ci.yml)
 
-```sh
-gh repo create --template freckle/haskell-library-template --public freckle/<name>
-```
+An `IdGenerator` and `Propagator` for [`hs-opentelemetry-sdk`][sdk] that
+generates and propagates `TraceId` and `SpanId` values that are [compatible with
+AWS X-Ray][xray].
 
-## Rename your package
+[sdk]: https://hackage.haskell.org/package/hs-opentelemetry-sdk
+[xray]: https://docs.aws.amazon.com/xray/latest/devguide/xray-api-sendingdata.html#xray-api-traceids
 
-```sh
-sed -i s/haskell-library-template/my-name/ ./**/*
-```
+## Usage
+
+The API currently exposed by the `hs-opentelemetry-sdk` package lacks a
```

Comment:
I'm sure I'm just missing context on this somewhere but what about upstreaming vs. adding a new package?

Reply:
Is it just that this is super XRay-specific so there's no natural home for it?

Reply:
Well, I guess what I meant to say was more like

> Until we go fix it, the API exposed by the...

Reply:
Added some language near this to indicate we should (and plan to) upstream something to make this unnecessary.